### PR TITLE
Telegram: add more detailed procedure

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Telegram: add more detailed procedure
- step-by-step to get started for first time users
- as the information is a bit dispersed in the official Telegram docs
- and not so easy to find.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/telegram.markdown [hunk 1]
```diff
@@ -11,15 +11,65 @@ ha_platforms:
...
+   * The bot will return your chat ID and the user name.
+1. Create a [Telegram bot in Home Assistant](/integrations/telegram_bot):
+   * Paste this into your configuration file:
+   * Replace the `api_key` and the `allowed_chat_ids` with your data.
+  
+      ```yaml
+      # Telegram Bot
+      telegram_bot:
+        - platform: polling
+          api_key: 1117774004:EABQulCACdgkQOTN3hS_5HZwSwxDlekCixr
```
Comment (frenck) [inline source/_integrations/telegram.markdown]:
```suggestion
          api_key: "1117774004:EABQulCACdgkQOTN3hS_5HZwSwxDlekCixr"
```

File: source/_integrations/telegram.markdown [hunk 2]
```diff
@@ -11,15 +11,65 @@ ha_platforms:
...
+
+1. Create a notifier:
+   * Paste this into your configuration file: 
+   * Replace the `name` and the `chat_id` with your data.
+  
+      ```yaml
+      # Notifier
+      notify:
+        - platform: telegram
+          name: sarah
```
Comment (frenck) [inline source/_integrations/telegram.markdown]:
```suggestion
          name: "sarah"
```

File: source/_integrations/telegram.markdown [hunk 3]
```diff
@@ -11,15 +11,65 @@ ha_platforms:
...
+1. In the chat with the new bot, enter `/start`.
+1. Test the service:
+   * Go to [**Developer tools** > **Services** > **YAML mode**](https://my.home-assistant.io/redirect/developer_call_service/?service=homeassistant.turn_on). 
+   * Paste this into the YAML file:
+   * Replace the `service` and the `message` with your data.
+  
+      ```yaml
+      service: notify.sarah
+      data:
+        message: Yay! A message from Home Assistant.
```
Comment (frenck) [inline source/_integrations/telegram.markdown]:
```suggestion
        message: "Yay! A message from Home Assistant."
```

File: source/_integrations/telegram.markdown [hunk 4]
```diff
@@ -11,15 +11,65 @@ ha_platforms:
...
+      ```
+   * Select **Call service**. You should now get a message.
```
Comment (frenck) [inline source/_integrations/telegram.markdown]:
```suggestion
   * Select **Call service**. You should now get a message.
```

File: source/_integrations/telegram.markdown [hunk 5]
```diff
@@ -11,15 +11,65 @@ ha_platforms:
...
+      notify:
+        - platform: telegram
+          name: sarah
+          chat_id: 44441111
+      ```
+
+1. From the conversation with BotFather, select the link to open a chat with your new bot.
+1. In the chat with the new bot, enter `/start`.
+1. Test the service:
+   * Go to [**Developer tools** > **Services** > **YAML mode**](https://my.home-assistant.io/redirect/developer_call_service/?service=homeassistant.turn_on).
```
Comment (frenck) [inline source/_integrations/telegram.markdown]:
```suggestion
   * Go to [**Developer tools** > **Services** > **YAML mode**](https://my.home-assistant.io/redirect/developer_call_service/?service=homeassistant.turn_on).
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]

**Comment by c0ffeeca7** [comment]
thank you!

**Comment by frenck** [comment]
No no, thank you! 👍

**Pull request merged**